source/_integrations/wilight.markdown

```diff
@@ -4,10 +4,12 @@ description: Instructions on how to integrate WiLight devices into Home Assistan
 ha_category:
   - Cover
   - Fan
+  - Irrigation
   - Light
+  - Switch
 ha_release: 0.115
 ha_config_flow: true
-ha_iot_class: Local Polling
+ha_iot_class: Local Push
```

Comment:
The integration manifest says it's Local Polling.

Reply:
Fixed!
Thanks.